### DISCUSSION TITLE
修改话题页面的一个顶栏

### DISCUSSION THREAD

**Discussion by CloudyEagle** [the1812/Bilibili-Evolved#5039] [功能建议 / Ideas]
若自定义顶栏打开背景模糊后，在话题页面往下翻看时会发现话题页会再弹出另一个顶栏，个人感觉展示出来的效果有点怪怪的。上面也就一个参加话题的按钮，希望脚本能做修改，我目前是用控制台将整个元素手动到隐藏掉达到下面内容动态透出，不过先用Adblock Plus将这个元素拦截。
话题页面链接大致为https://www.bilibili.com/v/topic/detail/?topic_id= 的页面吧
![image](https://github.com/user-attachments/assets/5f679017-53fd-4e8a-aa67-6ba15081b4fa)

![image](https://github.com/user-attachments/assets/c0697173-8f15-466b-9347-3e85371229e9)

![image](https://github.com/user-attachments/assets/1cfed96e-dd73-4c89-b9b8-af6a12f36eb3)

**Comment by Geeyun-JY3** [comment]
这种定制顶栏本脚本的适配方式应该是让`自定义顶栏`在此类页面忽略`全局固定`选项，露出原来的顶栏，先例可参考动态详情页。
